src/js/component.js

```diff
@@ -82,1182 +90,1103 @@ var Component = CoreObject.extend({
- * component without having to rewrite all the other default options.
- *
- *     Parent.prototype.options_ = {
- *       children: {
- *         'childOne': { 'foo': 'bar', 'asdf': 'fdsa' },
- *         'childTwo': {},
- *         'childThree': {}
- *       }
- *     }
- *     newOptions = {
- *       children: {
- *         'childOne': { 'foo': 'baz', 'abc': '123' }
- *         'childTwo': null,
- *         'childFour': {}
- *       }
- *     }
- *
- *     this.options(newOptions);
- *
- * RESULT
- *
- *     {
- *       children: {
- *         'childOne': { 'foo': 'baz', 'asdf': 'fdsa', 'abc': '123' },
- *         'childTwo': null, // Disabled. Won't be initialized.
- *         'childThree': {},
- *         'childFour': {}
- *       }
- *     }
- *
- * @param  {Object} obj Object of new option values
- * @return {Object}     A NEW object of this.options_ and obj merged
- */
-Component.prototype.options = function(obj){
-  if (obj === undefined) return this.options_;
+    return this.options_ = VjsUtil.mergeOptions(this.options_, obj);
+  }
 
-  return this.options_ = VjsUtil.mergeOptions(this.options_, obj);
-};
+  /**
+   * Get the component's DOM element
+   *
+   *     var domEl = myComponent.el();
+   *
+   * @return {Element}
+   */
+  el(){
+    return this.el_;
+  }
 
-/**
- * The DOM element for the component
- *
- * @type {Element}
- * @private
- */
-Component.prototype.el_;
+  /**
+   * Create the component's DOM element
+   *
+   * @param  {String=} tagName  Element's node type. e.g. 'div'
+   * @param  {Object=} attributes An object of element attributes that should be set on the element
+   * @return {Element}
+   */
+  createEl(tagName, attributes){
+    return Lib.createEl(tagName, attributes);
+  }
 
-/**
- * Create the component's DOM element
- *
- * @param  {String=} tagName  Element's node type. e.g. 'div'
- * @param  {Object=} attributes An object of element attributes that should be set on the element
- * @return {Element}
- */
-Component.prototype.createEl = function(tagName, attributes){
-  return Lib.createEl(tagName, attributes);
-};
-
-Component.prototype.localize = function(string){
-  var lang = this.player_.language(),
-      languages = this.player_.languages();
-  if (languages && languages[lang] && languages[lang][string]) {
-    return languages[lang][string];
+  localize(string){
+    var lang = this.player_.language(),
+        languages = this.player_.languages();
+    if (languages && languages[lang] && languages[lang][string]) {
+      return languages[lang][string];
+    }
+    return string;
   }
-  return string;
-};
 
-/**
- * Get the component's DOM element
- *
- *     var domEl = myComponent.el();
- *
- * @return {Element}
- */
-Component.prototype.el = function(){
-  return this.el_;
-};
+  /**
+   * Return the component's DOM element where children are inserted.
+   * Will either be the same as el() or a new element defined in createEl().
+   *
+   * @return {Element}
+   */
+  contentEl(){
```

Comment:
By default they point to the same, but some elements use sub elements to create their appearance and you don't want the the children to go into the main element anymore. MenuButtons might be one of those but I'm not remembering exactly.